internal/services/api/handlers/get_proofs_by_link.go

```diff
@@ -68,6 +70,21 @@ func GetLinkByID(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	var (
+		authorized                = len(UserClaim(r)) > 0
+		who                       = "Authorized(owner)"
+		fulfillVerificationEvents = false
+	)
+	if !authorized {
+		who = "Public"
```

Comment:
is this value will be used for something more then logging? if not, better to remove it. yes, then it requires more Log(r) method calls but additional redundant variable makes code less understandable

Reply:
reworked